ethereumj-core/src/main/java/org/ethereum/db/BlockStore.java

```diff
@@ -49,6 +49,17 @@
 
     List<Block> getListBlocksEndWith(byte[] hash, long qty);
 
+    /**
+     * Returns a list of blocks starting from toHash (inclusively) down to fromHash (inclusively).
+     * Both hashes should be of blocks belonging the same chain, otherwise method returns an empty list.
+     * If some blocks between from and to are missed then empty list is returned.
+     *
+     * @param fromHash lower bound of a list
+     * @param toHash upper bound of a list
+     * @return returns a list with inverted block order
+     */
+    List<Block> listBlocks(byte[] fromHash, byte[] toHash);
```

Comment:
I'd focus backwards nature of method in it's name or input param names

Reply:
Maybe it's just better to return blocks in direct order?

Reply:
If you don't need it in backwards order later, it's ok